### PR TITLE
CP-1355 Create COPYRIGHT_TRANSFER

### PR DESCRIPTION
**Pull request by sethladd**
Create COPYRIGHT_TRANSFER

### CODE REVIEW

**Comment by jayudey-wf** [comment]
thanks for putting this up so quickly!

File: COPYRIGHT_TRANSFER [hunk 1]
```diff
@@ -0,0 +1 @@
+I, Seth Ladd, hereby transfer copyright of dart_to_js_script_rewriter to Workiva Inc.
```
Comment (jayudey-wf) [inline COPYRIGHT_TRANSFER]: Sorry about this, I realize that in the example I sent you it said `transfer` but could you update that to be `assign`

**Comment by dustinlessard-wf** [comment]
+1

**Comment by evanweible-wf** [comment]
+1

**Comment by jayudey-wf** [comment]
QA +1, doc change only

Merging into master

**Pull request merged**